ppa/archive/gale.py

```diff
@@ -192,14 +208,19 @@ def get_item_pages(self, item_id, gale_record=None):
         # iterate through the pages in the response
         for page in gale_record["pageResponse"]["pages"]:
             page_number = page["pageNumber"]
```

Comment:
could just use this value directly instead of assigning to a local variable, now that we're only using it one place